install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -1,15 +1,21 @@
 SET NAMES 'utf8';
 
-INSERT INTO `PREFIX_configuration` (`name` , `value` , `date_add` , `date_upd`) VALUES ('PS_SMARTY_LOCAL', '0', NOW(), NOW());
-DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_ORDER_PROCESS_TYPE';
-DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_ADVANCED_PAYMENT_API';
-DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_ONE_PHONE_AT_LEAST';
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
+  ('PS_SMARTY_LOCAL', '0', NOW(), NOW()),
+  ('PS_PASSWD_RESET_VALIDITY', '1440', NOW(), NOW()),
+  ('PS_CUSTOMER_BIRTHDATE', '1', NOW(), NOW()),
+  ('PS_ACTIVE_CRONJOB_EXCHANGE_RATE', '0', '0000-00-00 00:00:00', '0000-00-00 00:00:00'),
```

Comment:
Can you put NOW() insteed '0000-00-00 00:00:00' 

![sql](https://cloud.githubusercontent.com/assets/9074610/19469388/6e839062-951b-11e6-80f8-8edb472abe30.png)